interfaces/data/IRssItem.ts

```diff
@@ -3,10 +3,14 @@
  */
 
 import type Parser from 'rss-parser';
-
+import { IRssPodcastValue } from './IRssPodcast';
 export interface IRssItem extends Parser.Item {
   [key: string]: any;
   itunes?: {
     [key: string]: any;
   };
+  podcast?: {
+    [key: string]: any;
+    value?: IRssPodcastValue;
```

Comment:
podcast:value can be an attribute of a feed or an item - I added parsing to both, but we're currently only using the feed's podcast:value tag.